x-pack/plugins/apm/public/components/shared/tutorial_fleet_instructions/index.tsx

```diff
@@ -0,0 +1,122 @@
+  align-items: center;
+`;
+
+type APIResponseType = APIReturnType<'GET /api/apm/fleet/has_data'>;
+
+function TutorialFleetInstructions({ http, basePath, isDarkTheme }: Props) {
+  const [data, setData] = useState<APIResponseType | undefined>();
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    async function fetchData() {
+      setIsLoading(true);
+      try {
+        const response = await http.get('/api/apm/fleet/has_data');
+        setData(response as APIResponseType);
+      } catch (e) {
+        console.error('Error while fetching fleet details.', e);
+      }
+      setIsLoading(false);
+    }
+    fetchData();
+  }, [http]);
+
+  if (isLoading) {
+    return (
+      <CentralizedContainer>
+        <EuiLoadingSpinner />
+      </CentralizedContainer>
+    );
+  }
+
+  // When APM integration is enable in Fleet
+  if (data?.hasData) {
+    return (
+      <EuiButton iconType="gear" fill href={`${basePath}/app/fleet#/policies`}>
+        {i18n.translate(
+          'xpack.apm.tutorial.apmServer.fleet.manageApmIntegration.button',
+          {
+            defaultMessage: 'Manage APM integration in Fleet',
+          }
+        )}
+      </EuiButton>
+    );
+  }
+  // When APM integration is not installed in Fleet or for some reason the API didn't work out
+  return (
+    <EuiPanel>
+      <EuiFlexGroup>
+        <EuiFlexItem grow={7}>
+          <EuiCard
+            display="plain"
+            textAlign="left"
+            title={i18n.translate('xpack.apm.tutorial.apmServer.fleet.title', {
+              defaultMessage: 'Elastic APM (beta) now available in Fleet!',
+            })}
+            description={i18n.translate(
+              'xpack.apm.tutorial.apmServer.fleet.message',
+              {
+                defaultMessage:
+                  'The APM integration installs Elasticsearch templates and Ingest Node pipelines for APM data.',
+              }
+            )}
+            footer={
+              <EuiButton
+                iconType="analyzeEvent"
+                color="secondary"
+                href={`${basePath}/app/fleet#/integrations/detail/apm-0.2.0/overview`}
+              >
+                {i18n.translate(
+                  'xpack.apm.tutorial.apmServer.fleet.apmIntegration.button',
+                  {
+                    defaultMessage: 'APM integration',
+                  }
+                )}
+              </EuiButton>
+            }
+          />
+        </EuiFlexItem>
+        <EuiFlexItem grow={3}>
+          <EuiImage
+            src={`${basePath}/plugins/apm/assets/${
+              isDarkTheme
+                ? 'illustration_integrations_darkmode.svg'
+                : 'illustration_integrations_lightmode.svg'
+            }`}
+            alt="Illustration"
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    </EuiPanel>
+  );
+}
+// eslint-disable-next-line import/no-default-export
+export default TutorialFleetInstructions;
```

Comment:
Would be great with a storybook for this - but not a must have for now

Reply:
yeah indeed. I'll do it in another PR, if that's ok with you, I can create one for the other component that shows the commands too.